From 87e6222b1c899562e5d44eba1d831d208ef56279 Mon Sep 17 00:00:00 2001
Message-Id: <87e6222b1c899562e5d44eba1d831d208ef56279.1683119947.git.luhux76@gmail.com>
From: Lu Hui <luhux76@gmail.com>
Date: Wed, 3 May 2023 21:15:49 +0800
Subject: [PATCH] gnu: gqrx: Update dependencies, use Qt 6.

* gnu/packages/radio.scm (gqrx)
[inputs]: Replace qtbase-5 with qt, qtsvg-5 with qtsvg.
---
 gnu/packages/radio.scm | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/gnu/packages/radio.scm b/gnu/packages/radio.scm
index 1f73c6171d..3d6be00bb8 100644
--- a/gnu/packages/radio.scm
+++ b/gnu/packages/radio.scm
@@ -13,6 +13,7 @@
 ;;; Copyright © 2022 Greg Hogan <code@greghogan.com>
 ;;; Copyright © 2022 Ryan Tolboom <ryan@using.tech>
 ;;; Copyright © 2023 Sharlatan Hellseher <sharlatanus@gmail.com>
+;;; Copyright © 2023 Lu Hui <luhux76@gmail.com>
 ;;;
 ;;; This file is part of GNU Guix.
 ;;;
@@ -1124,8 +1125,8 @@ (define-public gqrx
            log4cpp
            portaudio
            pulseaudio
-           qtbase-5
-           qtsvg-5
+           qtbase
+           qtsvg
            spdlog
            volk))
     (arguments

base-commit: 39ba8a10971f15264966823e8696d63c2995df86
prerequisite-patch-id: 64a8cee1d4f71df351da702af41ccd10f2c43070
prerequisite-patch-id: bc1b4e4fc28e5cb5b782edee3a9dce43a5f22163
prerequisite-patch-id: 9e023ac1d24979c5218d0b07558b3a1b98a0bf89
prerequisite-patch-id: 7deb1e21a838e06c79156f54d03c0886fdb8bc10
-- 
2.39.2

